### ISSUE TITLE
Elephpant

### ISSUE THREAD

**Issue by Guiraudou** [shopwareBoostDay/ask-me-anything#1]
Hi !
Probably not a question you are waiting for, but anyway, I can ask anything :p
Here is my question : is there a way to get the cody vuelette elephpant ?
Thanks,
Benoit

**Comment by ChristopherDosin** [comment]
Yeah - We need some Shopware SWAG. Where is the Shopware Store that existed a few years ago? ... :(
